### PR TITLE
Fix SickRage URL in comment and PEP rest of tests

### PR DESCRIPTION
**Pull request by labrys**
PEP 8: Optimize imports

### CODE REVIEW

File: tests/encoding_tests.py [hunk 1]
```diff
@@ -6,18 +6,17 @@
...
 import unittest
 
 sys.path.insert(1, os.path.abspath(os.path.join(os.path.dirname(__file__), '../lib')))
 sys.path.insert(1, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
 
 import sickbeard
+from sickbeard import ek, ex
 from sickrage.helper.common import sanitize_filename
-from sickrage.helper.encoding import ek
-from sickrage.helper.exceptions import ex
```
Comment (miigotu) [inline tests/encoding_tests.py:20]: this isnt pep8 =P, 1 import per line

File: tests/xem_tests.py [hunk 2]
```diff
@@ -17,23 +17,23 @@
...
+import unittest
 
 sys.path.insert(1, os.path.abspath(os.path.join(os.path.dirname(__file__), '../lib')))
 sys.path.insert(1, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
 
-import tests.test_lib as test
-import sickbeard
 from sickbeard.tv import TVShow
+import sickbeard
+import tests.test_lib as test
```
Comment (miigotu) [inline tests/xem_tests.py:36]: If you import test_lib first, you dont need to sys.path.insert here

Comment (labrys) [inline tests/xem_tests.py:36]: I had removed it locally and it broke running the tests individually, although all_tests still worked.

File: tests/encoding_tests.py (hunk 1 above)
Comment (labrys) [inline tests/encoding_tests.py:20]: Actually it is when doing selective imports from a single module.
https://www.python.org/dev/peps/pep-0008/#imports
=P

**Pull request merged**